nucleus/security/core/src/main/java/com/sun/enterprise/security/GroupPrincipal.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2021 Contributors to Eclipse Foundation. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+// Portions Copyright 2024 Payara Foundation and/or affiliates
```

Comment:
Portions copyright is only a thing with our usual license.
I believe for EPL you'd just add an entry for us below the "Contributors to Eclipse Foundation" line.

Reply:
As this is a direct port from GlassFish without changes, we don't need our own copyright.
```suggestion
```